packages/editor/src/server/index.tsx

```diff
@@ -42,6 +42,8 @@ async function asyncMain() {
             rel="stylesheet"
           />
           <meta name="viewport" content="width=device-width, initial-scale=1" />
+          <link rel="icon" type="image/png" sizes="64x64" href="../../assets/favicon-128.png" />
```

Comment:
that `assets` folder is controlled by webpack and should not contain any other files. But you can create a folder called `static` on that level. Please check `examples/website` for reference.